frontend/src/config.js

```diff
@@ -0,0 +1,3 @@
+const REACT_APP_TERM = "2022-T2";
```

Comment:
Maybe add these to `constants.js` instead as direct exports?

Reply:
Wait @banaboi I just noticed this - you should use `getlatestterm` and the `date` function to get the term and year, respectively